src/main/java/com/github/standobyte/jojo/action/non_stand/PillarmanWindCloak.java

```diff
@@ -40,13 +43,11 @@ public void stoppedHolding(World world, LivingEntity user, INonStandPower power,
     	windEffect(user, ModParticles.SANDSTORM.get(), 15);
     }
     
-    public static void windEffect(LivingEntity user, IParticleData particles, int intensity) {
-        for (int i = 0; i < intensity; i++) {
-            Vector3d particlePos = user.position().add(
-                    (Math.random() - 0.5) * (user.getBbWidth() + 0.5), 
-                    Math.random() * (user.getBbHeight()), 
-                    (Math.random() - 0.5) * (user.getBbWidth() + 0.5));
-            user.level.addParticle(particles, particlePos.x, particlePos.y, particlePos.z, Math.random() - 0.5, Math.random(), Math.random() - 0.5);
+    @Override
+    public void holdTick(World world, LivingEntity user, INonStandPower power, int ticksHeld, ActionTarget target, boolean requirementsFulfilled) {
+        if (!world.isClientSide() && requirementsFulfilled) {
+            user.addEffect(new EffectInstance(ModStatusEffects.FULL_INVISIBILITY.get(), 5, 0, false, false));
```

Comment:
The full invisibility effect causes the wind cloak layer to not render. It also has a couple of other problems, such as detectors not being able to detect the players with that effect, so it so better to not use it for now.
Maybe in that case it would be more appropriate to create a separate "Wind Cloak" effect, which makes it so the wind cloak renders, and the other effects do not (assuming that was the intention behind the change)

Reply:
seperated effect would be cool to a gameplay, and easier to implement it in a code. dikiytechies said that Yujin already did this "effect", so this part of this Pull Request is no longer suitable for the latest dev-version =(